docs/layers.md

```diff
@@ -1,116 +1,510 @@
-# Layers in the vector tile service
+# Layers in Mapzen's vector tiles
 
-Data is organized into several layers comprising the elements typically used for base map rendering. This is a simplified view of OSM data for easier consumption, with common tags often condensed into a single field as noted below.
+![image](images/mapzen-vector-tile-docs-all-layers.png)
 
-## Water
-Polygons representing oceans, riverbanks and lakes. Derived from a combination of the `waterway`, `natural`, and `landuse` tags. Includes coastline-derived water polygons from [openstreetmapdata.com](http://openstreetmapdata.com) at higher zoom levels, and [Natural Earth](http://naturalearthdata.com) polygons at lower zoom levels.
+The [Mapzen vector tile service](https://mapzen.com/projects/vector-tiles) provides worldwide basemap coverage sourced from [OpenStreetMap](www.openstreetmap.org) and other open data projects, updated daily as a free & shared service.
 
-Layer name: `water`
+Data is organized into several layers, grouped into an `all` layer comprising the elements typically used for base map rendering. This is a simplified view of OpenStreetMap data for easier consumption, with common tags often condensed into a single `kind` field as noted below.
 
-Properties:
+Need help displaying vector tiles in a map? We have several [examples](display-tiles.md) using Mapzen vector tiles to style in your favorite graphics library including Tangram, Mapbox GL, D3, and Hoverboard.
+
+## Overview
+
+#### Data sources and attribution
+
+Mapzen primarily sources from OpenStreetMap, but includes a variety of other open data. For a full listing, view the [data sources](data-sources.md). Each source may require [attribution](attribution.md) in your project.
+
+#### Name localization
+
+Mapzen vector tile features include all source names in the default `name` property, and variants in the `name:*` properties. When different than `name`, we also include `alt_name:*`, `alt_name_`, `old_name:*` values to enable full internationalization. Tangram supports all scripts.
+
+For features in the Boundaries layer, we support two additional variants `left:name:*` and `right:name:*` to support oriented labeling on the appropriate side of the boundary line (so the labeled polygon's text can appear inside that polygon consistently).
```

Comment:
Is it worth having a consistent markup for referring to layers? In some places we quote them, in others use `code` markup, and sometimes treat them as words. Here, "Boundaries" is capitalized like a name, but elsewhere it isn't.